python/vowpalwabbit/DFtoVW.py

```diff
@@ -513,6 +569,94 @@ def process(self):
         return "".join(out)
 
 
+class _ListLabel(object):
+    """An helper class that handles a list of labels.
+
+    The class is to be used in the `DFtoVW` class. It parses the list of labels
+    that the user could provide in the `label` argument of `DFtoVW`. The class only accepts
+    a pre-defined set of label classes (defined in the class attribute `available_labels`).
+
+    Parameters
+    ----------
+    label_list : list
+        The list of labels that the user passed in the attribute label of DFtoVW.
+
+    Raises
+    ------
+    ValueError
+        If the list passed has mixed types or if the labels should not be used in a list.
+    """
+
+    available_labels = (ContextualbanditLabel, MultiLabel)
+    sep_by_label = dict(ContextualbanditLabel=" ", MultiLabel=",")
+
+    def __init__(self, label_list):
+
+        instance_classes = set([type(label_instance).__name__ for label_instance in label_list])
+        if len(instance_classes) > 1:
+            raise TypeError("The list passed in 'label' has mixed label types.")
+
+        if not all(
+            isinstance(label_instance, self.available_labels) for label_instance in label_list
+        ):
+            raise TypeError(
+                "The only labels that can be used with list are {accepted}.".format(
+                    accepted=repr([cls.__name__ for cls in self.available_labels])[1:-1]
+                )
+            )
+
+        # Unpack columns
+        columns = set()
+        for label in label_list:
+            columns.update(label.columns)
+        self.columns = columns
```

Comment:
This step is necessary since the attribute `columns` is used in `DFtoVW.check_missing_columns_df`